features/lwipstack/lwipopts.h

```diff
@@ -339,7 +339,9 @@
 #define LWIP_ETHERNET               0
 #endif // MBED_CONF_LWIP_ETHERNET_ENABLED
 
-#define LWIP_L3IP   				0
+#define LWIP_L3IP                   0
```

Comment:
Why `LWIP_L3IP` is this still disabled?

Reply:
According to design LWIP_L3IP is disabled by default. It is planned to be enabled via json config file

Reply:
Maybe this should be then changed as:

```
#if MBED_CONF_LWIP_L3IP_ENABLED
#define LWIP_L3IP                   1
#else
#define LWIP_L3IP                   0
#endif
```

Reply:
And l3ip-enabled option to be added to lwip's mbed_lib.json.

Reply:
Done